portfolio/src/main/webapp/assets/css/images/poptrox-nav.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="96px" height="64px" viewBox="0 0 96 64" zoomAndPan="disable">
+

```

Comment:
not sure if this was auto added or not (if it was automatic, don't worry about it) but these spaces can probably be removed